src/lamplib/__main__.py

```diff
@@ -101,15 +102,19 @@ def main():
         elif args.subcommand == "benchmark-test":
             tasks.run_tests.benchmark_test(compile_env)
         elif args.subcommand == "resmoke-test":
-            tasks.run_tests.resmoke_test(
+            success = tasks.run_tests.resmoke_test(
```

Comment:
If an exception gets raised in this function, it should just percolate up and cause a non-zero return anyway, right?

Reply:
The resmoke test failure as a subprocess call doesn't actually throw an exception, just has its own return code. I'd be cool with testing and throwing inside if that's preferred